src/etc/inc/filter.inc

```diff
@@ -422,11 +422,31 @@ function filter_configure_sync($delete_states_if_needed = true) {
 		$mt = microtime();
 		echo "pfctl being called at $mt\n";
 	}
-	unset($rules_loading, $rules_error);
-	$_grbg = exec("/sbin/pfctl -o basic -f {$g['tmp_path']}/rules.debug 2>&1", $rules_error, $rules_loading);
-	if (isset($config['system']['developerspew'])) {
-		$mt = microtime();
-		echo "pfctl done at $mt\n";
+	
+	$pfctl_try = 0;
+	while ($pfctl_try < 10) {
+		$pfctl_try++;
```

Comment:
It has the same net effect, but using `for` is preferred for a loop with a set number of iterations. It would combine these three lines into one and be easier to read:
`for ($pfctl_try = 0; $pfctl_try < 10; $pfctl_try++) {`